JitsiConference.js

```diff
@@ -1340,8 +1340,11 @@ JitsiConference.prototype.onIncomingCall
 
     // TODO: remove deprecated cross region property from this specific event
     // once all existing deployments include analytics changes
-    if (window.jitsiDeploymentInfo) {
-        crossRegion = window.jitsiDeploymentInfo.CrossRegion;
+    if (this.options.config.deploymentInfo) {
```

Comment:
I'm sorry I didn't see your last reply... It's fine the way it is. Here's an alternative way that I've used a lot (and can be used with destructuring) when I don't have a util library handy or don't want to create my own util:
```javascript
const obj2 = obj1 && obj1.obj2;
const obj3 = obj2 && obj3;
```
The main difference is you don't have to access obj1.obj2.obj3 at the end.